bin/mock_onfido.pl

```diff
@@ -0,0 +1,334 @@
+# Automatically enables "strict", "warnings", "utf8" and Perl 5.10 features
+use Mojolicious::Lite;
```

Comment:
Do we use this script for anything else other than running tests?  If so, it might be better situated under `t/` instead.

Reply:
At least it will be used to test bom-rpc & bom-events. Maybe it will be a service like `mock-experian` (not sure).